CONTRIBUTING.md

```diff
@@ -24,6 +24,29 @@ pre-installed dependencies, running web server and rollup watching for changes.
 
 [![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/from-referrer/)
 
+### Making a PR through the online-setup
+
+- Fork the repo by pressing `F1` and then type in fork and then hit enter.
+
+- You can commit your changes either using a the `git` cli or using the GUI in the top left tab
+  under search as shown below:
+
+  ![image](https://user-images.githubusercontent.com/46004116/76298550-6fb95c00-62db-11ea-9fda-f4e8c0840e69.png)
+
+- Once you are done commiting your changes you can push the code via using push button from the
+  GitHub tab in the top right corner as shown:
+
+  ![image](https://user-images.githubusercontent.com/46004116/76299985-c9228a80-62dd-11ea-930c-00eb22156c9e.png)
+
+- Once you have made all the changes and pushed them to your fork. You can make PR from the same
+  GitHub tab in top right corner as shown:
+
+  ![image](https://user-images.githubusercontent.com/46004116/76300458-9036e580-62de-11ea-8f24-ebaa664204ce.png)
```

Comment:
Is it possible to have this commit message pre-populated with our pull request template?  To make sure user's have met all the requirements for the PR?

Reply:
Thanks! Not yet! :) but there is a proposal for this from one of the team members https://github.com/gitpod-io/gitpod/issues/1045

Reply:
One that issue is fixed/integrated, I think we can merge this PR